persistence/sql/persister_client.go

```diff
@@ -76,13 +76,21 @@ func (p *Persister) DeleteClient(ctx context.Context, id string) error {
 	return sqlcon.HandleError(p.Connection(ctx).Destroy(&client.Client{ID: cl.ID}))
 }
 
-func (p *Persister) GetClients(ctx context.Context, limit, offset int) ([]client.Client, error) {
+func (p *Persister) GetClients(ctx context.Context, filters client.ClientFilters) ([]client.Client, error) {
 	cs := make([]client.Client, 0)
-	return cs, sqlcon.HandleError(
-		p.Connection(ctx).
-			Paginate(offset/limit+1, limit).
-			Order("id").
-			All(&cs))
+
+	query := p.Connection(ctx).
+		Paginate(filters.Offset/filters.Limit+1, filters.Limit).
+		Order("id")
+
+	if filters.Name != "" {
+		query.Where("client_name = ?", filters.Name)
+	}
+	if filters.Owner != "" {
+		query.Where("owner = ?", filters.Owner)
+	}
```

Comment:
This probably means that we need to index these fields? Otherwise this will get very slow when you have lots of clients. Then again, if we just have like 1000 clients, it probably wouldn't matter too much (maybe 5ms with index, 200ms without). What do you think?

Reply:
For our use case, the index won't make much of a difference

Reply:
Ok, we can keep the filter omitted for now and add it later.